packages/api/helper/nominationGsheetToDB.js

```diff
@@ -67,18 +67,22 @@ module.exports = function gsheetToDB() {
               parseFloat(nomination[37].replace(/\$/g, '')) * 100
             ),
           },
-        }).then((array) => {
-          //db.Nomination.findOrCreate returns a promise that when fulfilled returns an array with two elements 
-          //the first element is the nomination instance object that contains the dataValues among other things  
-          //the second element is a boolean that represents whether a nomination was created
-          if (array[1]) {
-            verifyHcEmail(array[0].dataValues)
+        });
+        if (array[1]) {
+          let nom = await db.Nomination.findOne({
+            where: {
+              emailValidated: true,
+              providerEmailAddress: nomination[12],
+            },
```

Comment:
Should we set `array[0].emailValidated` to `true` in the DB if the email is already validated on another nom?

Reply:
Conditional has been added to address this.